packages/dds/matrix/test/testconsumer.ts

```diff
@@ -3,8 +3,9 @@
  * Licensed under the MIT License.
```

Comment:
Opportunistically updated TestConsumer to use the RowMajorMatrix from Tiny-Calc rather than roll its own.

The one "real" change is that it now reads the cell values during row/col changed notifications, which triggers the bug fixed by this PR.